### DISCUSSION TITLE
Any chance of a U1->UT2004 workflow?

### DISCUSSION THREAD

**Discussion by dserious** [xtremexp/UT4X-Converter#85] [Ideas]
Is there any chance a Unreal 1 to UT2004 map conversion workflow is on the roadmap?

**Comment by dserious** [comment]
Bump :)

**Reply by dserious** [reply]
@xtremexp

**Comment by dserious** [comment]
FWIW -- the U1 -> UT2004 Workflow fails in the UT3 Converter II:

`
History: PrepBrush <- ImportProperties <- ImportActorProperties <- Finishing <- ULevelFactory::FactoryCreateText <- UFactory::StaticImportObject <- UEditorEngine::Exec_Map <- UEditorEngine::Exec <- (MAP IMPORT FILE="C:\Users\dserious\OneDrive\Desktop\UT3converter2-025c\Converted).. <- UUnrealEdEngine::Exec <- WEditorFrame::OnCommand <- WWindow::WndProc <- WWindow::StaticProc <- MessagePump <- MainLoop

`

**Comment by xtremexp** [comment]
For U1 to UT2004 workflow, please UT3 Converter. Since i haven't updated it in a while it might no longer work and it's no longer maintained for now.
